### PR TITLE
#168058121 User gets all bookmarks

### PR DESCRIPTION
**Pull request by kennyedward**
#### What does this PR do?
This pull request implements user get all bookmarks functionality

#### Description of Task to be completed
- Users get all articles added to a bookmark

#### How should this be manually tested?
 - Clone this repository
 - cd into the repo & run ```npm install```
- Using Postman **SET** ```Authorization``` to ```Bearer <user token>```
- Make a **GET** request to /api/v1/users/bookmarks
#### Any Background Context You want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168058121](https://pivotaltracker.com/story/show/168058121)

#### Screenshots (if appropriate)
All bookmarks found
No bookmark

#### Questions:
N/A

### CODE REVIEW

**Comment by iverenshaguy** [comment]
Good work here. We don't want an error message for a user with 0 bookmarks, we want an empty array instead since no error occurred. We also don't want `currentCount` as part of our response, instead we want the `page` (number) we are on and the `count` of data returned.  Please update. Thanks

File: src/controllers/user.js [hunk 1]
```diff
@@ -173,5 +173,25 @@ export default {
...
+   * @param {Object} response - express response object
+   *
+   * @returns {Object} - all bookmark for a user
+   */
+  getAllBookmark: async (request, response) => {
+    const { id: userId, limit = 10, page = 1 } = request.user;
+
+    const data = await paginator(Bookmarks, { where: { userId }, page, limit });
+
+    return response
```
Comment (iverenshaguy) [inline src/controllers/user.js]: Where does the error in your screenshot come in?

**Review by iverenshaguy — CHANGES_REQUESTED** [review]

File: src/controllers/user.js (hunk 1 above)
Comment (kennyedward) [reply src/controllers/user.js]: From the paginator function that was implemented by a colleague. From its implementation, immediately there's no count, it throws a NotFoundError() 
```helpers/paginator``` line 29

**Comment by kennyedward** [comment]
> Good work here. We don't want an error message for a user with 0 bookmarks, we want an empty array instead since no error occurred. We also don't want `currentCount` as part of our response, instead we want the `page` (number) we are on and the `count` of data returned. Please update. Thanks

I have updated the code as requested.

File: src/controllers/user.js (hunk 1 above)
Comment (iverenshaguy) [reply src/controllers/user.js]: I think this has been fixed. Please confirm.

**Pull request merged**